clients/src/main/java/org/apache/kafka/clients/consumer/internals/HeartbeatRequestManager.java

```diff
@@ -553,7 +553,7 @@ public ConsumerGroupHeartbeatRequestData buildRequestData() {
                 }
             } else {
                 // SubscribedTopicRegex - only sent if has changed since the last heartbeat
-                //                      - not supported yet
+
```

Comment:
Why not go ahead and implement the `SentFields` code and call `data.setSubscribedTopicRegex()`?